src/evidently/test_preset/data_drift.py

```diff
@@ -145,4 +168,17 @@ def generate_tests(self, data: InputData, columns: DatasetColumns):
             )
         )
 
+        if embeddings_data is not None:
```

Comment:
same here. also, the looks like a good candidate for code de-duplication: create utility function?